### ISSUE TITLE
Logging question

### ISSUE THREAD

**Issue by gmartinez** [codegreencreative/laravel-samlidp#10]
Hey there, question, I have the debug flag turned on, but none is being written in the .log file.

I am basically looking to get the .xml being exchanged in the requests/responses, any way better than other to do that ?

**Comment by upwebdesign** [comment]
@gmartinez yes, you are correct. The SamlidpLog trait does not appear to be implemented at all. If you want to try to give it a stab and create a pull request, that would be great. Otherwise I will label this as an enhancement.

**Comment by gmartinez** [comment]
I can try do that indeed :)

---
Gonzalo Martinez | In-Prove IT Services <http://www.in-prove.com> | UY Tel:
+598 99 260393
